service/src/main/kotlin/io/provenance/api/domain/cee/ContractService.kt

```diff
@@ -14,7 +14,7 @@ import java.util.UUID
 
 interface ContractService {
     fun getContract(contractName: String): Class<out P8eContract>
-    fun <T : P8eContract> setupContract(client: Client, contractClass: Class<T>, records: Map<String, Message>, scopeUuid: UUID, sessionUuid: UUID? = null, participants: Map<Specifications.PartyType, Originator>? = null, scope: ScopeResponse? = null, scopeSpecification: String, audiences: Set<PublicKey>): Session
+    fun <T : P8eContract> setupContract(client: Client, contractClass: Class<T>, records: Map<String, Message>, scopeUuid: UUID, sessionUuid: UUID? = null, participants: Map<Specifications.PartyType, KeyEntity>? = null, scope: ScopeResponse? = null, scopeSpecification: String, audiences: Set<PublicKey>): Session
```

Comment:
😱 Where's the linting at?
```suggestion
    fun <T : P8eContract> setupContract(
        client: Client, contractClass: Class<T>,
        records: Map<String, Message>,
        scopeUuid: UUID,
        sessionUuid: UUID? = null,
        participants: Map<Specifications.PartyType, KeyEntity>? = null,
        scope: ScopeResponse? = null,
        scopeSpecification: String,
        audiences: Set<PublicKey>,
    ): Session
```

Reply:
:hmmm: :truestory: